client/packages/system/src/Item/DetailView/Tabs/ItemVariants/ItemVariantsTab.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useState } from 'react';
+
+import {
+  AppBarButtonsPortal,
+  AppFooterPortal,
+  BasicTextInput,
+  ButtonWithIcon,
+  DialogButton,
+  FlatButton,
+  InputWithLabelRow,
+  NumericTextInput,
+  Typography,
+} from '@common/components';
+import {
+  Box,
+  useTranslation,
+  PlusCircleIcon,
+  EditIcon,
+} from '@openmsupply-client/common';
+import { ItemPackagingVariantsTable } from './ItemPackagingVariantsTable';
+import { ItemVariantFragment, PackagingVariantFragment } from '../../../api';
+
+const dummyPackVariants: PackagingVariantFragment[] = [
+  {
+    __typename: 'PackagingVariantNode',
+    id: '1',
+    packagingLevel: 1,
+    name: 'Primary',
```

Comment:
I was thinking we might have a mapping something like `label.packaging_level.1` = `Primary` so it was translatable, and potentially could still allow the user to still set a name, but I think this is good. We might need some way to default to this when we add new variants I guess?

Reply:
Yep, my thinking when implementing the `add` button properly was to have these translatable - hadn't thought about them being able to change the name though - I suppose as long as we display the packaging level then we could!